src/assets/templates/typescript/tslint.json

```diff
@@ -4,6 +4,22 @@
       "tslint-config-airbnb"
   ],
   "jsRules": {},
-  "rules": {},
+  "rules": {
+    "max-line-length": [
```

Comment:
✅ This makes good sense because we often go over this (especially on step expressions.